GNUmakefile

```diff
@@ -65,8 +69,8 @@ count ?= 1
 
 
 # Test rules:
-test: $(GO-DEPS)
-	go test$(if $v, -v) -vet=off .
+test: $(GO-DEPS) $(V3-DIR)
```

Comment:
I have no concerns about this change, but it looks like it may be pointing to version-specific test runs in the future. Is that the case?

Reply:
It's just using a stable version of our YAML decoder for driving the yaml based tests.
Bootstrapping...